Tasks/codepush-patch/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "vsts-task-code-push-patch",
+  "version": "1.0.0",
+  "description": "Patch an app update on the CodePush service",
```

Comment:
Let's match the CLI's description here: "Update the metadata for an existing release"